packages/shared/components/inputs/AssetAmountInput.svelte

```diff
@@ -36,7 +36,7 @@
         unit = Unit.i
     }
 
-    export function validate(): Promise<void> {
+    export function validate(): Promise<void | string> {
```

Comment:
Why would this return a string? If you want it to return a string, then let it always return a string, otherwise let it always return void